### PR TITLE
MNT: update recipe

### PR DESCRIPTION
**Pull request by st3107**
New release.

https://github.com/xpdAcq/PDFstream/releases/tag/0.3.2

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -44,10 +43,10 @@ test:
     - pdfstream.vend
     - pdfstream.visualization
   commands:
-    - pdfstream --help
+    - pdfstream -- --help
 
 about:
-  home: https://github.com/st3107/pdfstream
+  home: https://github.com/{{ org }}/pdfstream
```
Comment (mrakitin) [inline recipe/meta.yaml:49]: I'd recommend to put the org name explicitly here, so we can just copy the URL.

Comment (mrakitin) [reply recipe/meta.yaml:49]: Also, I'd suggest keeping the original, case-sensitive version: https://github.com/xpdAcq/PDFstream.

File: recipe/conda_build_config.yaml [hunk 2]
```diff
@@ -8,9 +8,5 @@ cuda_compiler_version:
   - None
 docker_image:
   - nsls2/nsls2forge:latest
-numpy:
-  - 1.16
-python:
-  - 3.7
```
Comment (mrakitin) [inline recipe/conda_build_config.yaml:14]: Why were these lines removed?

File: recipe/LICENSE [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+2. Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.
+
+3. Neither the name of the copyright holder nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
```
Comment (mrakitin) [inline recipe/LICENSE:13]: This file does not seem right. As I explained in another PR's review, it should be either https://github.com/xpdAcq/PDFstream/blob/master/LICENSE, or just removed from this repo, as it will be picked up from the released sources.

File: recipe/meta.yaml (hunk 1 above)
Comment (st3107) [reply recipe/meta.yaml:49]: Done.

File: recipe/conda_build_config.yaml (hunk 2 above)
Comment (st3107) [reply recipe/conda_build_config.yaml:14]: Add back.

File: recipe/LICENSE (hunk 3 above)
Comment (st3107) [reply recipe/LICENSE:13]: Fixed.
